via/views/view_pdf.py

```diff
@@ -67,6 +67,14 @@ def proxy_google_drive_file(request):
     return _iter_pdf_response(request.response, content_iterable)
 
 
+@view_config(route_name="proxy_jstor_pdf", decorator=(has_secure_url_token,))
+def proxy_jstor_pdf(request):
+    content_iterable = request.find_service(JSTORAPI).jstor_pdf_stream(
+        request.params["doi"]
```

Comment:
I think we should pass this as part of path as other pdf enpoints in via do. Then the only query param will be the original URL.